README.md

```diff
@@ -1,3 +1,29 @@
-FastAPI
-Postgres
-Docker
+
+# FastAPI with Postgres
+- FastAPI app
+- PostgreSQL with SQLAlchemy
+- Python-Decouple
+- Pydantic
+
+## Docker
+```Hello user$ docker-compose up --build```
+
+## Run
+```poetry run uvicorn api.main:app --reload```
+
+
+## Postgres
+Run postgres: ```sudo -u postgres psql```
+
+Show databases: ```\l```
+
+Use databases: ```\c```
+
+Show tables: ```\dt```
+
+Restart postgres server: ```sudo service postgresql restart```
```

Comment:
start with docker, restart with docker 😹 https://docs.docker.com/engine/reference/commandline/restart/ .
Btw if you want to restart and have an empty postgres it is just about removing volume with "docker compose down -v" and starting again "docker compose up"